python/mxnet/contrib/quantization.py

```diff
@@ -758,11 +745,6 @@ def calib_graph(qsym, arg_params, aux_params, collector,
     """User-level API for calibrating a quantized model using a filled collector.
     The backend quantized operators are only enabled for Linux systems. Please do not run
     inference using the quantized models on Windows for now.
-    The quantization implementation adopts the TensorFlow's approach:
-    https://www.tensorflow.org/performance/quantization.
-    The calibration implementation borrows the idea of Nvidia's 8-bit Inference with TensorRT:
-    http://on-demand.gputechconf.com/gtc/2017/presentation/s7310-8-bit-inference-with-tensorrt.pdf
-    and adapts the method to MXNet.
```

Comment:
Keep this notes here for future reference.

Reply:
file too long (>1000L):(